x-pack/plugins/observability_solution/observability_ai_assistant/README.md

```diff
@@ -50,24 +42,200 @@ The user can then also continue the conversation in a flyout by clicking "Start
 
 - **1.2.2. Action Menu Button**
 
-All Observability apps also have a button in the top action menu, to open the AI Assistant and start a conversation.
+When navigating through Observability apps, there is an Assistant menu in the top button. Click it to start a new conversation or access existing ones.
 
 - **1.2.3. Standalone page**
 
 Users can also access existing conversations and create a new one by navigating to `/app/observabilityAIAssistant/conversations/new`. They can also find this link in the search bar.
 
 #### **1.3. Chat**
 
-Conversations with the AI Assistant are powered by three foundational components: the LLM (currently only OpenAI flavors), the knowledge base, and function calling.
+Conversations with the AI Assistant are powered by three foundational components: the LLM, the knowledge base, and function calling.
 
-The LLM essentially sits between the product and the user. Its purpose is to interpret both the messages from the user and the response from the functions called, and offer its conclusions and suggest next steps. It can suggest functions on its own, and it has read and write access to the knowledge base.
+The LLM essentially sits between the platform and the user. Its purpose is to interpret both the messages from the user and the response from the functions called, and offer its conclusions and suggest next steps. It can suggest functions on its own, and it has read and write access to the knowledge base.
 
 The knowledge base is an Elasticsearch index, with an inference processor powered by ELSER. Kibana developers can preload embeddings into this index, and users can access them too, via plain Elasticsearch APIs or specific Kibana APIs. Additionally, the LLM can query the knowledge base for additional context and store things it has learned from a conversation.
```

Comment:
I think we should make it clear why semantic search is useful in this context over traditional lexical search.

```suggestion
The knowledge base is an Elasticsearch index, with an inference processor powered by ELSER. This enables us to query the knowledge base with the users original question, using semantic search.
Kibana developers can preload embeddings into this index, and users can access them too, via plain Elasticsearch APIs or specific Kibana APIs. Additionally, the LLM can query the knowledge base for additional context and store things it has learned from a conversation.
```